backend/functions/read_stats.py

```diff
@@ -0,0 +1,32 @@
+'''
+Function to retrieve numeric values on label from the converted string.
+Parameters: convertedImg -> string (converted image)
+Returns: tuple -> calories, fat, carbohydrates, and sugar in grams
+'''
+
+
+def getFromImage(convertedImg):
+    checkString = convertedImg.lower()
+    checkString = checkString.replace(" ", "")
+    calIndex = checkString.index("calories") + 8
+    calories = ""
+    while checkString[calIndex].isnumeric():
+        calories += checkString[calIndex]
+        calIndex += 1
+    fatIndex = checkString.index("totalfat") + 8
```

Comment:
Will totalfat always be the label? Likewise for totalcarbohydrate. What happens if the food label does not have one of the labels?  I think trying to cast an empty string to an integer will throw an exception, so we might have to be careful for that case.